pkg/host/machine_info_linux.go

```diff
@@ -140,3 +144,63 @@ func getModulesInfo() ([]KernelModule, error) {
 	}
 	return modules, nil
 }
+
+func getDirsInfo(globs []string) (map[string][]string, error) {
+	excludes := []string{
+		"/proc/sysrq-trigger",
```

Comment:
Uh, this is ugly. I would suggest disabling CONFIG_MAGIC_SYSRQ in the kernel config. Whatever way the fuzzer reaches it, it will do bad things.